### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by StayBlue** [sssata/HYKU_CAD#7]
There's no words or comments on the status of Linux support for this driver in the README, so I assume it is unsupported. Maybe that would be good to add to the README that it's not supported yet?

**Comment by YourSandwich** [comment]
Also would be interessted, maybe it could be integrated into OpenTabletDriver which has an native linux version?

**Comment by sssata** [comment]
@StayBlueee 
You're right, there is no Linux support for now, I'll add that to the documentation.

However since the configurator program uploads the settings to the permanent storage of the tablet, you can try uploading the settings on a Windows PC and then using it with a different Linux PC. You don't need to have the HYKU_Configurator.exe running to use the tablet.

@YourSandwich 
Yes I've thought about doing that though I haven't really found the time to emulate a tablet in firmware yet.

**Comment by YourSandwich** [comment]
Maybe you could integrate the configurator into OpenTabletDriver as a plugin for example so you don't need to emulate it as a tablet.
